crates/cargo-test-macro/src/lib.rs

```diff
@@ -133,8 +133,18 @@ pub fn cargo_test(attr: TokenStream, item: TokenStream) -> TokenStream {
                     "rustup or stable toolchain not installed"
                 );
             }
-            s if s.starts_with("requires_") => {
+            s if s.starts_with("requires=") => {
                 let command = &s[9..];
+                let Ok(literal) = command.parse::<Literal>() else {
+                    panic!("expect a string literal, found: {command}");
+                };
+                let literal = literal.to_string();
+                let Some(command) = literal
```

Comment:
should be able to call `.value()`, see https://github.com/clap-rs/clap/blob/b4ea2d4be9f01b6c7945084cbd158d6075cf2408/clap_derive/src/item.rs#L1390

Reply:
No? Because we are not using `syn` :(